Tests/SwiftLintFramework/TestHelpers.swift

```diff
@@ -151,6 +151,14 @@ extension XCTestCase {
         ruleDescription.corrections.forEach(config.assertCorrection)
         // make sure strings that don't trigger aren't corrected
         zip(nonTriggers, nonTriggers).forEach(config.assertCorrection)
+
+        //"disable" command do not correct
+        ruleDescription.corrections.forEach { ( before, _) in
```

Comment:
This is more of a personal style preference, but I like to omit the parentheses in these cases: `{ before, _ in`